cmd/internal/cli/actions.go

```diff
@@ -185,7 +185,7 @@ func replaceURIWithImage(ctx context.Context, cmd *cobra.Command, args []string)
 		image, err = handleOras(ctx, imgCache, cmd, origImageURI)
 	case uri.Shub:
 		image, err = handleShub(ctx, imgCache, origImageURI)
-	case oci.IsSupported(t):
+	case ocitransport.SupportedTransport(t):
```

Comment:
Given the package name (which I assume is intended to stay `ocitransport`, would it make sense to keep this as IsSupported() (or Supported()) so that it doesn't stutter?

Reply:
I'm going to leave this as-is here... the `ocitransport` package is a temporary thing to break an import cycle that otherwise occurs across client and build code. It should go away as the #1022 work continues.